kedro/extras/transformers/README.md

```diff
@@ -0,0 +1,6 @@
+# Information regarding transformers in Kedro
+
+# Time profiler
+This section outlines the prupose and functionality of the "time profiler" as a transformer. 
+By importing "AbstractTransformer", time profiler enables any contributors to track and log the running time for loading any dataset, as well as saving any calls.

```

Comment:
Can you replace "contributors" with "users". :)

Reply:
I think this section could do with a little more detail, and maybe even an example? "saving any calls" isn't immediately clear imo